RockLib.Logging/LogEntry.cs

```diff
@@ -178,6 +179,13 @@ public LogLevel Level
     /// </param>
     public void SetExtendedProperties(object? extendedProperties)
     {
+#if NETCOREAPP2_0_OR_GREATER || NET5_0_OR_GREATER
```

Comment:
We shouldn't need any checks for .NET version anymore since we updated to only use 4.8, 3.1(which we'll be removing soon) and 6.0.

Reply:
Well, in this case, are we saying that we only want this done for .NET 6 apps, and not .NET 4.8?

Reply:
The `System.Diagnostics.Activity` class is not available in .NET 4.8 which is why this check is here.

Reply:
But we don't target .NET Core 2.0 or .NET 5, so should this change to looking for .NET Core 3.1 and .NET 6 (and greater for each)?

Reply:
This if statement does not have any affect on what framework is supported.  It only tells the compiler to use the block if it matches one of those statements.

Reply:
The `System.Diagnostics.Activity` class only exists in Core 2.0 and up and .NET 5 and up which is why those are the specifications being made here.